scss/detail.scss

```diff
@@ -42,3 +47,8 @@
   height: 368px;
   width: 261px;
 }
+
+.book-separator {
+  width: 540px;
```

Comment:
There is no need to set a width here. You could set `width: 100%` to your `book-info` container like so:

![image](https://user-images.githubusercontent.com/36488701/68690927-a069ce80-0551-11ea-8c32-cc36d4ff46bf.png)